pybossa.js

```diff
@@ -689,20 +690,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         return ajaxResponse;
     };
 
-    pybossa.savePartialAnswerToServer = function (taskId, data) {
-        const url = URL + 'api/task/' + taskId + '/partial_answer';
+    pybossa.savePartialAnswerToServer = function (taskId, data, projectName) {
+        const url = `${URL}api/project/${projectName}/task/${taskId}/partial_answer`;
         let ajaxResponse = _postRequest(url, JSON.stringify(data));
         return ajaxResponse;
     }
 
-    pybossa.getSavedAnswerFromServer = function (taskId) {
-        const url = URL + 'api/task/' + taskId + '/partial_answer';
+    pybossa.getSavedAnswerFromServer = function (taskId, projectName) {
+        const url = `${URL}api/project/${projectName}/task/${taskId}/partial_answer`;
         let ajaxResponse = _getRequest(url);
         return ajaxResponse;
     }
 
-    pybossa.deleteSavedAnswerFromServer = function (taskId) {
-        const url = URL + 'api/task/' + taskId + '/partial_answer';
+    pybossa.deleteSavedAnswerFromServer = function (taskId, projectName) {
+        const url = `${URL}api/project/${projectName}/task/${taskId}/partial_answer`;
         let ajaxResponse = _deleteRequest(url);
         return ajaxResponse;
     };
```

Comment:
In https://github.com/bloomberg/pybossa/pull/826/files, we have a new endpoint `/project/<short_name>/has_partial_answer`. Is this not called by pybossa.js?

Reply:
That's right. It's meant to be used in the next sprint's UI ticket. The idea is that, before a user clicking "Start Contributing Now!" button, the page should know whether the user has any saved tasks or not. If there is a saved task, the user will have the option to decide whether they want to retrieve the saved tasks first or last.